src/proc/formats-converter.cpp

```diff
@@ -183,6 +183,12 @@ void formats_converter::prepare_to_convert( stream_profiles from_profiles )
             {
                 if( factory_of_best_match->has_source( raw_profile ) )
                 {
+                    // When using DDS the server may split one stream of data into multiple dds_streams, e.g. Y8I
+                    // infrared data. When grouping these dds_streams under one sensor we get multiple instances of the
+                    // same profile, but the server should only open one to streaming
+                    if( is_profile_in_list( raw_profile, { current_resolved_reqs.begin(), current_resolved_reqs.end() } ) )
```

Comment:
When changing the implementation, so that
1. No two converters give the same output profile (as is the case now for Y8/Y8I profiles for index 1)
2. No interlaced format will be sent
then this code could be removed

Reply:
So you'll remove in the next PR?

Reply:
Yes, in the next PR I will verify it is no longer needed and remove

Reply:
Removed in [PR#11946](https://github.com/IntelRealSense/librealsense/pull/11946)